docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+case "$1" in
+    "init")
+        blueoil init "${@:2}"
+        ;;
+    "train")
+        blueoil train "${@:2}"
+        ;;
+    "convert")
+        blueoil convert "${@:2}"
+        ;;
+    "predict")
+        blueoil predict "${@:2}"
+        ;;
```

Comment:
IMO: how about unifying and passing through?
```
"init" | "train" | "convert" | "predict")
    blueoil "$@"
    ;;
```

Reply:
YES, THANKS!!

Reply:
IMO: and better to add cases for showing blueoil command help messages: `""`, `"--help"` 
Because current `blueoil` command shows help messages when without arg or with `--help` 👍

Reply:
Super good idea!!